apps/web/app/helpers/date.ts

```
@@ -58,6 +58,27 @@ export function secondsToTime(secs: number) {
+ * This function takes two Date objects, `startedAt` and `stoppedAt`, and computes
+ * the difference between them in seconds. If either of the dates is invalid,
+ * the function returns `undefined`.
+ *
+ * @param {Date} startedAt - The starting time
+ * @param {Date} stoppedAt - The stopping time
+ * @returns {number | undefined} The difference in seconds or `undefined` if dates are invalid
+ */
+export function differenceBetweenHours(startedAt: Date, stoppedAt: Date): number {
+	const started = new Date(startedAt);
+	const stopped = new Date(stoppedAt);
+	if (!isNaN(started.getTime()) && !isNaN(stopped.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)